client/my-sites/plugins/plugins-browser/style.scss

```diff
@@ -93,9 +95,14 @@ body.is-section-plugins #primary .main.is-logged-out {
 
 		.responsive-toolbar-group__button-item {
 			padding: 4px 12px;
-		}
-		.responsive-toolbar-group__button-item:not([class*="is-pressed"]):hover::before {
-			background: rgba(6, 117, 196, 0.1);
+
+			&::before {
+				background-color: transparent;
+			}
+
+			&:not([class*="is-pressed"]):hover::before {
+				background-color: color-mix(in srgb, var(--studio-blue) 10%, transparent);
```

Comment:
This is definitely a vast improvement over what was there, hard-coded, before. I'm guessing the reason we can't go component-first in this case is it's a PHP page? 

@jameskoster PHP components keep coming up, and we know for sure we'll need some. I'm wondering: should we create a single stylesheet that lightly reskins these PHP elements in one central place, so we can reduce local styles? And ryelle would that be useful here or not? Not a blocker in any case, mainly thinking ahead.

Reply:
> This is definitely a vast improvement over what was there, hard-coded, before. I'm guessing the reason we can't go component-first in this case is it's a PHP page?

The component has a grey hover color, but the logged out plugin (and theme) showcase pages use this light blue color. Maybe it was one of the screenshots that didn't load, let me try again. Here, the hover state is the lighter blue.

![Screenshot 2025-04-01 at 2 56 24 PM 6 19 26 PM](https://github.com/user-attachments/assets/dfd1c8d8-b218-4f1a-afb3-390365bcc7e0)

This also isn't a PHP page, we're in Calypso 🙂